src/Models/BaseElement.php

```diff
@@ -5,7 +5,9 @@
 use DNADesign\Elemental\Controllers\ElementController;
 use DNADesign\Elemental\Forms\TextCheckboxGroupField;
 use DNADesign\Elemental\ORM\FieldType\DBObjectType;
+use DNADesign\Elemental\TopPage\TopPageTrait;
```

Comment:
```suggestion
use DNADesign\Elemental\TopPage\TopPageTrait;
```
Can we please no longer use the `TopPage` dir, it's always bugged me that it's literally the only things that uses a 'feature' based dir structure.  Instead could you please delete the TopPage dir and instead move thing things inside dir to match the rest our conventions e.g. create a 'Traits' dir and move TopPageTrait there instead.  You'll need to update namespaces accordingly.